bufit_test.go

```diff
@@ -11,7 +11,7 @@ import (
 	"time"
 )
 
-func ExampleBytes() {
+func exampleBytes() {
```

Comment:
You're right, but they already weren't being shown because the go vet issue (Bytes is an unknown identifier). I'll send a cl to fix this and actually expose this example.